plugin/src/main/java/net/dzikoysk/funnyguilds/listener/region/BucketAction.java

```diff
@@ -1,12 +1,12 @@
 package net.dzikoysk.funnyguilds.listener.region;
 
 import net.dzikoysk.funnyguilds.feature.protection.ProtectionSystem;
+import net.dzikoysk.funnyguilds.listener.AbstractFunnyListener;
 import org.bukkit.event.EventHandler;
-import org.bukkit.event.Listener;
 import org.bukkit.event.player.PlayerBucketEmptyEvent;
 import org.bukkit.event.player.PlayerBucketFillEvent;
 
-public class BucketAction implements Listener {
+public class BucketAction extends AbstractFunnyListener {
```

Comment:
This class does not benefit from `AbstractFunnyListener`, so there is no point to use it I guess

Reply:
just in case